src/ucp/core/ucp_ep.c

```diff
@@ -302,7 +305,7 @@ ucs_status_t ucp_ep_init_create_wireup(ucp_ep_h ep,
                                                   HW TM is supported */
     key.am_bw_lanes[0]        = 0;
     key.rma_lanes[0]          = 0;
-    key.rma_bw_lanes[0]       = 0;
+    key.rma_bw_lanes[0]       = UCP_NULL_LANE;
```

Comment:
is it really needed? rndv should not start before getting the RTS active message anyway

Reply:
yes, ucp_worker_get_ep_config fails with assertion for ud[x] TLS which does not support RMA

Reply:
ok